src/test/groovy/GenerateBuildDataIntegrationTests.groovy

```diff
@@ -114,6 +114,15 @@ class GenerateBuildDataIntegrationTests {
     assertFalse(obj.get("test").isEmpty())
     assertFalse(obj.get("build").isEmpty())
     assertFalse(obj.get("env").isEmpty())
+
+    // Then metadata is removed
+    assertNull(obj.get("changeSet")[0].author?._class)
+    assertNull(obj.get("changeSet")[0].author?._links)
+
+    // Then some duplicated entries don't exist anymore
+    assertNull(obj.get("build.branch"))
+    assertNull(obj.get("build.changeSet"))
+    assertNull(obj.get("build.pullRequest"))
```

Comment:
Those entries are already defined in the `job` field:

![image](https://user-images.githubusercontent.com/2871786/98944247-abe73100-24e8-11eb-9156-f3f97dba2e4c.png)